seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java

```diff
@@ -16,8 +16,9 @@
  */
 
 package org.apache.seatunnel.common.constants;
+import java.io.Serializable;
 
-public enum JobMode {
+public enum JobMode implements Serializable{
```

Comment:
I think this change is useless. Please refer https://docs.oracle.com/javase/1.5.0/docs/guide/serialization/spec/serial-arch.html#enum